k8s/kafka/operator/deployment-script.sh

```
@@ -1,7 +1,7 @@
 #!/bin/bash
-wget https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.42.0/strimzi-0.42.0.tar.gz
-tar -zxvf strimzi-0.42.0.tar.gz
-cd  strimzi-0.42.0/
+wget https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.45.0/strimzi-0.45.0.tar.gz
+tar -zxvf strimzi-0.45.0.tar.gz
+cd  strimzi-0.45.0/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)